App/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Tourney:wght@400;500;600;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./main.css">
+    <title>Simón Dice</title>
+</head>
+
+<body>
+    <header>
+        <h1>Simón Dice</h1>
+        <p>Repite la secuencia de luces del tablero y <b>GANA</b></p>
+    </header>
+    <main class="contain-game">
+        <div id="verde" class="verde" data-color="verde"></div>
+        <div id="rojo" class="rojo" data-color="rojo"></div>
```

Comment:
Cuando no se le envía componentes `children` a los tags, podrías simplificarlo de esta forma
```suggestion
        <div id="rojo" class="rojo" data-color="rojo" />
```

Reply:
ah esto pense que en html no aplicaba, creía que era algo que se podía hacer con react.

Reply:
Lo probé y la verdad que no funciona de esa manera. así que lo dejo como estaba, con la etiqueta de cierre

Reply:
Reforzando el comentario: https://stackoverflow.com/questions/10347844/wondering-why-self-closing-div-does-not-work
La confusión es esperable porque estamos acostumbrados a escribir código en archivos JSX en React, que mayormente es similar a HTML pero usa algunas reglas de XML https://es.reactjs.org/docs/introducing-jsx.html